src/test/java/ua/cv/tim/service/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,201 @@
+package ua.cv.tim.service.impl;
+
+
+import org.mockito.*;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import ua.cv.tim.dao.AllianceDao;
+import ua.cv.tim.dao.PlayerDao;
+import ua.cv.tim.dao.UserDao;
+import ua.cv.tim.dto.UserDTO;
+import ua.cv.tim.model.Alliance;
+import ua.cv.tim.model.Player;
+import ua.cv.tim.model.Role;
+import ua.cv.tim.model.User;
+import ua.cv.tim.utils.SendMail;
+
+import javax.mail.MessagingException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+import static org.testng.Assert.*;
+
+
+/**
+ * Created by mmaksymtc on 24.01.2017.
+ */
+
+public class UserServiceImplTest {
+
+    @Mock
+    UserDao userDao;
+    @Mock
+    AllianceDao allianceDao;
+    @Mock
+    PlayerDao playerDao;
+    @Mock
+    SendMail sendMail;
+    @InjectMocks
+    UserServiceImpl userService;
+    @Captor
+    ArgumentCaptor<User> captor;
+    @Spy
+    List<User> users = new ArrayList<User>();
+
+    @BeforeClass
+    public void setUp(){
+        MockitoAnnotations.initMocks(this);
+        users=getUserList();
+    }
+    @AfterMethod
+    public void resetAllMocks(){
+        Mockito.reset(allianceDao,playerDao,userDao,sendMail);
+    }
+
+    @Test
+    public void getUserByUsernameTest() {
```

Comment:
naming convention - word 'test' in the begining of method name - testSomeLogic()